### PR TITLE
Hotfixes in the database & project files logic into develop

### PR DESCRIPTION
**Pull request by valentynfaychuk**
Repaired project files & corresponding databases creation and removal when opening & closing project.

### CODE REVIEW

File: src/mainwindow/mainwindow.cpp [hunk 1]
```diff
@@ -1002,26 +1002,64 @@ void MainWindow::onNewProjectTriggered()
         return;
     }
 
-    databaseConnect(dirName);
+    databaseConnect( );
```
Comment (valentynfaychuk) [inline src/mainwindow/mainwindow.cpp]: remove space from inside braces

File: src/mainwindow/mainwindow.cpp [hunk 2]
```diff
@@ -1002,26 +1002,64 @@ void MainWindow::onNewProjectTriggered()
...
-    qDebug()<<"on databaseConnect function";
+    setDatabaseFileName();
+    db = ConnectionGetter::getDefaultConnection(databaseFileName);
     CreateDB database;
-    database.addTableFile();
+    /*database.addTableFile();
     database.addTableUser();
     database.addTableComment();
-    database.addTableMessage();
+    database.addTableMessage();*/
```
Comment (valentynfaychuk) [inline src/mainwindow/mainwindow.cpp]: Remove comments from code

**Review by valentynfaychuk — COMMENTED** [review]
Consider comments

**Review by opastushkov — APPROVED** [review]

File: src/filemanager/filemanager.cpp [hunk 3]
```diff
@@ -91,3 +91,9 @@ bool FileManager::sourceFileByTheSameNameExists(const QString &headerName)
 {
     return (QFileInfo::exists(removeExtension(headerName,2).append(".cpp")));
 }
+
+QString FileManager::projectFileName = "";
```
Comment (PeterLysyk) [inline src/filemanager/filemanager.cpp:95]: better use QString() instead of ""

File: src/mainwindow/mainwindow.h [hunk 4]
```diff
@@ -120,6 +124,10 @@ public slots:
 
 private:
     friend class SettingsConfigurator;
+    QString  databaseFileName;
+    QString dirName;
+    const char *projectDatabaseExtension = ".db";
```
Comment (PeterLysyk) [inline src/mainwindow/mainwindow.h:129]: better create const literal

**Review by PeterLysyk — APPROVED** [review]

**Pull request merged**